controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -103,16 +106,9 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, err
 	} else if isTerminating {
 		reqLogger.Info("MCO instance is in Terminating status, skip the reconcile")
-		svmCrdExists, err := util.CheckCRDExist(r.CrdClient, config.StorageVersionMigrationCrdName)
-		if err != nil {
-			return ctrl.Result{}, err
-		}
-		if svmCrdExists {
-			// remove the StorageVersionMigration resource and ignore error
-			cleanObservabilityStorageVersionMigrationResource(r.Client, instance)
-		}
```

Comment:
the above section is added to cleanup `StorageVersionMigration` resource. we should keep it. that is a rebase issue.

Reply:
@clyang82 
this logic move to initFinalization() func
Since we don't need to clean cert-mgr related stuff now. So this one is the only remaining thing we need to handle in finalize step